src/wasm/request.rs

```diff
@@ -87,7 +88,7 @@ impl Request {
 }
 
 impl RequestBuilder {
-    pub(super) fn new(client: Client, request: crate::Result<Request>) -> RequestBuilder {
+    pub fn new(client: Client, request: crate::Result<Request>) -> RequestBuilder {
```

Comment:
Let's keep this private. Particularly, that it is created from an initial `Result` is an internal detail that should stay that way.

Reply:
Done 😄